docs/framework/30scoring/overview.md

```diff
@@ -2,8 +2,8 @@
 The overall risk score is calculated as the product of the risk scores of each of the categories:
 
 ```
-Overall Risk Score = Network Consensus Risk x Architectural Risk x Implementation Risk x Operational Risk
+Overall Risk Score = 100,000 - (Network Consensus Risk x Architectural Risk x Implementation Risk x Operational Risk)
```

Comment:
This solves the issue outlined in #92, for the _overall risk score_. However, it looks like the subscores still have that problem?

Reply:
Correct. The individual scores are high is low risk. 

I created a draft, trying to do the opposite approach. What I found was that numbers needed to go from 1 to 10, and also have infinity. This allowed for the fact that one aspect might have extremely high risk, that outweighed all other aspects of the scoring. Trying to explain this I think is more confusing that the current approach.